runtime/commonMain/src/kotlinx/serialization/internal/Tuples.kt

```diff
@@ -78,7 +85,8 @@ public class MapEntrySerializer<K, V>(
     override fun toResult(key: K, value: V): Map.Entry<K, V> = MapEntry(key, value)
 }
 
-// todo: move from internal package and add documentation
+@InternalSerializationApi
+@Deprecated(message = deprecationMessage, level = DeprecationLevel.HIDDEN)
```

Comment:
Forgot `ReplaceWith`?

Reply:
This one is hidden

Reply:
It still be marked as deprecated on fq-name import